### ISSUE TITLE
[Toggle] Color specs update

### ISSUE THREAD

**Issue by aweell** [dxc-technology/halstack-style-guide#307]
## React

```
focusColor: globalTokens.blue
selectedBackgroundColor: globalTokens.purple
selectedBackgroundHoverColor: globalTokens.black
selectedFontColor: globalTokens.white
selectedHoverFontColor: globalTokens.white
unselectedBackgroundColor: globalTokens.lightGrey
unselectedBackgroundHoverColor: globalTokens.darkWhite
unselectedFontColor: globalTokens.black
unselectedHoverFontColor: globalTokens.black
disabledSelectedBackgroundColor: globalTokens.lighterPurple
disabledSelectedFontColor: globalTokens.white
disabledUnselectedBackgroundColor: globalTokens.lightWhite
disabledUnselectedFontColor: globalTokens.lighterBlack
```

## Angular

```
"--toggle-unselectedBackgroundColor": globalTokens.lightGrey,
"--toggle-unselectedBackgroundHoverColor": globalTokens.darkWhite,
"--toggle-unselectedFontColor": globalTokens.black,
"--toggle-unselectedHoverFontColor": globalTokens.black,
"--toggle-selectedBackgroundColor": globalTokens.purple,
"--toggle-selectedHoverBackgroundColor": globalTokens.black,
"--toggle-selectedFontColor": globalTokens.white,
"--toggle-selectedHoverFontColor": globalTokens.white,
"--toggle-disabledSelectedBackgroundColor": globalTokens.lighterPurple,
"--toggle-disabledSelectedFontColor": globalTokens.white,
"--toggle-disabledUnselectedBackgroundColor": globalTokens.lightWhite,
"--toggle-disabledUnselectedFontColor": globalTokens.lighterBlack,
```

## Toggle color specs table

| Component token                     | Element                     | Token                    | Value       |
| :---------------------------------- | :-------------------------- | :----------------------- | :---------- |
| `unselectedBackgroundColor`         | Container fill:enabled	    | `color-grey-200`         | `#e6e6e6`     |
| `unselectedHoverBackgroundColor`    | Container fill:hover	    | `color-grey-300`         | `#cccccc`     |
| `unselectedActiveBackgroundColor`   | Container fill:active	    | `color-purple-700`       | `#5f249f`     |
| `unselectedDisabledBackgroundColor` | Container fill:disabled	    | `color-grey-100`         | `#f2f2f2`     |
| `unselectedFontColor`               | Label	                    | `color-black`            | `#000000`     |
| `unselectedDisabledFontColor`       | Label:disabled	            | `color-grey-500`         | `#999999`     |
| `selectedBackgroundColor`           | Container fill:enabled	    | `color-purple-700`       | `#5f249f`     |
| `selectedHoverBackgroundColor`      | Container fill:hover	    | `color-purple-800`       | `#4b1c7d`     |
| `selectedActiveBackgroundColor`     | Container fill:active	    | `color-purple-900`       | `#321353`     |
| `selectedDisabledBackgroundColor`   | Container fill:disabled	    | `color-purple-100`       | `#f2eafa`     |
| `selectedFontColor`                 | Label	                    | `color-white`            | `#ffffff`     |
| `selectedDisabledFontColor`         | Label:disabled	            | `color-purple-300`       | `#cbacec`     |
| `focusColor`                        | Border:focus	            | `color-blue-600`         | `#0095ff`     |

**Comment by mnnaveenk** [comment]
## React

focusColor: globalTokens.blue
selectedBackgroundColor: globalTokens.purple
selectedBackgroundHoverColor: globalTokens.black
selectedFontColor: globalTokens.white
selectedHoverFontColor: globalTokens.white
unselectedBackgroundColor: globalTokens.lightGrey
unselectedBackgroundHoverColor: globalTokens.darkWhite
unselectedFontColor: globalTokens.black
unselectedHoverFontColor: globalTokens.black
disabledSelectedBackgroundColor: globalTokens.lighterPurple
disabledSelectedFontColor: globalTokens.white
disabledUnselectedBackgroundColor: globalTokens.lightWhite
disabledUnselectedFontColor: globalTokens.lighterBlack

## Angular

"--toggle-unselectedBackgroundColor": globalTokens.lightGrey,
"--toggle-unselectedBackgroundHoverColor": globalTokens.darkWhite,
"--toggle-unselectedFontColor": globalTokens.black,
"--toggle-unselectedHoverFontColor": globalTokens.black,
"--toggle-selectedBackgroundColor": globalTokens.purple,
"--toggle-selectedHoverBackgroundColor": globalTokens.black,
"--toggle-selectedFontColor": globalTokens.white,
"--toggle-selectedHoverFontColor": globalTokens.white,
"--toggle-disabledSelectedBackgroundColor": globalTokens.lighterPurple,
"--toggle-disabledSelectedFontColor": globalTokens.white,
"--toggle-disabledUnselectedBackgroundColor": globalTokens.lightWhite,
"--toggle-disabledUnselectedFontColor": globalTokens.lighterBlack,

## Toggle color specs table



|   Component token                       |   Element                 |   Core token                      |   Value               |
| --------------------------------------- | ------------------------- | --------------------------------- | --------------------- |
|   `focusColor`                          |   container:focus         |  `globalTokens.blue`              |   `#005FCC`           |
|   `selectedBackgroundColor`             |   contaner                |  `globalTokens.purple`            |   `#6f2c91`           |
|   `selectedBackgroundHoverColor`        |   container: Hover        |  `Hal-Black`                      |   `#000000`           |
|   `selectedHoverBackgroundColor`        |   container: Hover        |  `Hal-Black`                      |   `#000000`           |
|   `selectedHoverFontColor`              |   container: Hover        |  `Hal-White`                      |   `#ffffff`           |
|   `selectedFontColor`                   |   container text          |  `Hal-White`                      |   `#ffffff`           |
|   `unselectedBackgroundColor`           |   container               |  `globalTokens.lightGrey`         |   `#D9D9D9`           |
|   `unselectedBackgroundHoverColor`      |   container: Hover        |  `globalTokens.darkWhite`         |   `#eeeeee`           |
|   `unselectedHoverFontColor`            |   container: Hover        |  `Hal-Black`                      |   `#000000`           |
|   `unselectedFontColor`                 |   container: text         |  `Hal-Black`                      |   `#000000`           |
|   `disabledSelectedBackgroundColor`     |   container               |  `globalTokens.lighterPurple`     |   `#f5ebf9`           |
|   `disabledUnselectedBackgroundColor`   |   container               |  `Hal-Grey-L-95`	                |   `#f2f2f2`           |
|   `disabledSelectedFontColor`           |   container text          |  `Hal-White`                      |   `#ffffff`           |
|   `disabledUnselectedFontColor`         |   container text          |  `globalTokens.lighterBlack`      |   `#b1b1b1`           |

**Comment by GomezIvann** [comment]
Changes made on the Toggle Group advanced tokens:

- [ ] `selectedBackgroundHoverColor` renamed as `selectedHoverBackgroundColor`.
- [ ] `unselectedBackgroundHoverColor` renamed as `unselectedHoverBackgroundColor`.

**Comment by aweell** [comment]
Closed via #502
